tools/reports/phone/calls.js

```diff
@@ -76,6 +87,7 @@ function getCallsList (backup) {
 function getCallsListiOS7 (backup) {
   return new Promise((resolve, reject) => {
     // Attempt to open database.
+    // iOS 6 and earlier versions do not explicitly store the type of service.
```

Comment:
This is not a concern, as we only support iOS 14 as the oldest version :)

You can check the supported versions here: https://www.qustodio.com/en/downloads/